### ISSUE TITLE
hi can you give a train log?

### ISSUE THREAD

**Issue by zhangyunming** [avBuffer/UNet3plus_pth#1]
i train the v3 with my own data that have 0:backgrounbd  and 1(portrait), but the loss is unchange.

**Comment by zhangyunming** [comment]
the bug is ok.its similar with pytorch-unet

**Comment by avBuffer** [comment]
hey, you maybe change into UNet2plus to try. If the error is same, there is wrong in your data or using loss function. Else there is wrong in UNet3plus model structure.

**Comment by avBuffer** [comment]
Hey,thanks! I tried it and found the training loss was changed after about 18 steps. In fact,  from step-0 to step-17, the training loss had a bit a bit changed. You maybe train a long steps.

**Comment by jialeqaq** [comment]
I train myself on data, DICE is low and constant, how do I solve it

**Comment by jialeqaq** [comment]
> i train the v3 with my own data that with lables 0:backgrounbd and 1:portrait, just two classes ,so the n_classes=1 , but you masks labels are 1-255 , and my train loss is not change and the test result are all black 0 . so how can i do .thanks

hello，why my training data DICE is very low and the more I train, the worse it gets
